src/signals/IncidentMap/components/IncidentMap/IncidentMap.tsx

```diff
@@ -66,6 +66,19 @@ export const IncidentMap = () => {
     resetMarkerIcons()
   }, [resetMarkerIcons])
 
+  const toggleFilter = useCallback(
+    (categoryName: Filter[], checked: boolean) => {
```

Comment:
by just having the name you can toggle the checked..so checked should be passed down the tree of components. Then do a find on it and return the result.